z-open/zerv-core
# 5534 - ET - Fix inactive timeout overflow preventing from users to log in.

User sessions cannot last longer than 90 days (hard coded)

<h3 id="steps-to-reproduce">1. Steps to Reproduce</h3>

**Test library**
Make sure the automatic tests pass.

`npm test`

**Test for regression your application login and logout functionalities**
The test cases are in the app pr 5534. 
This is the idea:
- log in your app and open many tabs
- Log out
- All tabs should be logged out
- log in your app and open many tabs
- Wait for your session timeout
- All tabs should be logged out

<h3 id="performance-considerations">2. Performance Considerations</h3>
No performance impact besides adding minimum timeout triggering logic.

<h3 id="additional-notes">Additional Notes</h3>

Zerv Sync and security libs  are also impacted due to new Zerv core lib version.
